pygfx/objects/_volume.py

```diff
@@ -0,0 +1,40 @@
+from ._base import WorldObject
+from ..geometries import BoxGeometry
+from ..resources import Buffer
+
+
+class Volume(WorldObject):
```

Comment:
Didn't we agree this would be a subclass of `Mesh`? I'm a little surprised to see it being a subclass of `Volume`. Did you need special treatment in the renderer?

Reply:
The renderer is not the limitation, because we can trigger on the material. But semantically it makes sense (to me) that this is not a mesh. Because it is not? Unless you consider anything that has geometry a mesh.

Reply:
If I construct a Mesh manually (pass it a correct BoxGeometry and VolumeSliceMaterial), will I get the same visual result?

Reply:
At the moment, the renderer only triggers on a `VolumeSliceMaterial` on a `Volume`.

My view is that you can create a `Mesh`, and then use texcoords (as we do in e.g. `multislice1.py`. Then you do more work, because you are responsible for positioning the mesh. Alternatively, you use a `Volume`, in which case the geometry is implicit (created automatically).

I think these underpin a different way of thinking. In the first case you say: I'm going to put a mesh here in my scene, and on it's surface I am going to project a slice through a volume (using 3D texture coords). In the second case you say: I'm going to put a volume here in my scene, and I am going to view just a slice through it (using the VolumeSliceMaterial, as opposes to e.g. a VolumeMipMaterial).

So yes, you get the same visual result, but I don't think it warrants that a Volume is a Mesh subclass. Anyway, that's my current view; we can talk :)

Reply:
I can live with it, it doesn't matter that much. :)